src/eXpansion/Bundle/Menu/Model/Menu/ChatCommandItem.php

```diff
@@ -14,7 +14,7 @@
  * @copyright 2017 eXpansion
  * @package eXpansion\Bundle\Menu\Model\Menu
  */
-class ChatCommandItem extends AbstractItem
+class ChatCommandItem extends AbstractItem implements ItemInterface
```

Comment:
AbstractItem implements ItemInterface already ???

Reply:
phpStorm didn't understand this for some reason.. it showed syntax error...

Reply:
Which version are you using? I have no issue with this. And having it on each Item is actually really not top.